lib/networking/kustomization.yaml

```diff
@@ -0,0 +1,367 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+resources:
+- metallb_deploy.yaml
+- metallb_ipaddresspools.yaml
+- metallb_l2advertisement.yaml
+- nmstate_deploy.yaml
+- ocp_nodes_nncp.yaml
+- netattach_ctlplane.yaml
+- netattach_internalapi.yaml
+- netattach_storage.yaml
+- netattach_tenant.yaml
+- netconfig.yaml
+
+patches:
+- target:
+    kind: NodeNetworkConfigurationPolicy
+    labelSelector: "osp/nncm-config-type=standard"
```

Comment:
This is intended. 

We originally added it so people could run this during [stage 3](https://github.com/openstack-k8s-operators/architecture/tree/main/va/hci/stage3).

```
oc wait nncp -l osp/interface=enp7s0 --for condition=available --timeout=300s
```
Of course the interface name could change though and we'd have to have kustomize change labels. So instead we just have have a `labelSelector` for the "standard NNCM-config" so that regardless of name people can just run this:
```
oc wait nncp -l osp/nncm-config-type=standard --for condition=available --timeout=300s
```
I updated the docs too (hci.md for now).

Reply:
Thanks for the explanation @fultonj !